cardinal/option.go

```diff
@@ -28,7 +28,7 @@ type Option func(*World)
 //
 // NOTE: WithAdapter is automatically injected when Cardinal is running in production mode.
 // This option should only be used for testing, or to stub in your own adapter implementation.
-func WithAdapter(adapter shard.Adapter) WorldOption {
+func WithAdapter(adapter adapter.Adapter) WorldOption {
 	return WorldOption{
```

Comment:
I personally don't care for this, but why is stuttering ok here from your pov.

Reply:
its common in go to have a package `foo` that provides type `foo`. theres not really ways to avoid this. the stuttering thing becomes an issue for unnecessary stuttering, which is why i renamed `AdapterConfig` to just `Config`.